packages/eyes/src/index.ts

```diff
@@ -58,6 +81,20 @@ async function runTests(url: string) {
   }
 }
 
+function logEyesResult({name, status, isNew, appUrls}: any) {
+  const isError = status === 'Unresolved' && isNew.toString() === 'false';
+  const dividerString = '--------------------';
+  const errorString = '~~~~~~~~~~~~~~~~~~~~';
+  const divider = isError ? chalk.red(errorString) : dividerString;
+
+  consoleLog(`
+  ${divider}
+  ${isError ? chalk.red('ERROR') : ''} ${chalk.bold(name)}: ${status === 'Unresolved' ? chalk.red(status) : chalk.green(status)}. Is new: ${isNew.toString() === 'false' ? chalk.green(isNew) : chalk.yellow(isNew)}.
+  ${chalk.cyan('URL')}: ${chalk.underline(appUrls.batch)}
```

Comment:
We don't need to print the batch URL for each failed test, let's print it once at the bottom.